test/ContentHubLoggingClientTest.php

```diff
@@ -23,6 +24,8 @@
  */
 class ContentHubLoggingClientTest extends TestCase {
 
+  use ProphecyTrait;
```

Comment:
@saxumVermes are we using this trait?

Reply:
Yes, this is required to avoid deprecation notices. Prophesize function is defined there.